addons/project/models/project_task.py

```diff
@@ -12,6 +12,7 @@
 from odoo.exceptions import UserError, ValidationError, AccessError
 from odoo.osv import expression
 from odoo.tools.misc import get_lang
+from odoo.addons.resource.models.utils import filter_domain_leaf
```

Comment:
apparently this is not used, so you should remove it
sorry I just noticed :sweat_smile: